scss/navbar.scss

```diff
@@ -0,0 +1,26 @@
+$nav-height: 90px;
+$text-big: 24px;
+$wolox-logo-size: 200px;
+$grey: #C4C4C4;
+
+.navbar {
+  display: flex;
+  align-content: flex-start;
+  justify-content: space-between;
+  height: $nav-height;
+  font-size: $text-big;
+  font-family: $roboto;
+  padding: 30px 20px;
+  box-shadow: 0 8px 5px -5px $grey;
+  width: 100%;
+  top: 0;
+  background-color: white;
+  position: fixed;
+  z-index: 1;
+
+  &::before {
```

Comment:
usually the logo inside the navbar is an anchor to the home page so it's not advisable to do this as a pseudo element